pkg/reconcilermanager/controllers/reconciler_container_log_level.go

```diff
@@ -107,7 +107,9 @@ func mutateContainerLogLevel(c *corev1.Container, override []v1beta1.ContainerLo
 			switch c.Name {
 			case metrics.OtelAgentName:
 				// otel-agent surfaces the log level configuration differently.
-				zapLevel := zapcore.Level(logLevel.LogLevel)
+				// Our log levels range from 0-10, whereas zap ranges from -1 (debug) to 5 (fatal).
```

Comment:
i thought 5 was info and fatal was higher?

Reply:
This comment is referring to the underlying integer mapping of the zap library

Reply:
See https://pkg.go.dev/go.uber.org/zap/zapcore#Level

Reply:
oh, i see. you set anything >= 6 to "debug" (aka -1)